firefox/guide/firefox/firefox_preferences-development_tools/oval/firefox.xml

```diff
@@ -0,0 +1,24 @@
+<def-group>
+  <definition class="compliance" id="firefox_preferences-development_tools"  version="1">
+    <metadata>
+      <title>Firefox Development Tools Must Be Disabled.</title>
```

Comment:
Please make the title:
`Disable Background Information Submission`

Reply:
Edited the title accordingly